ZenPacks/zenoss/OpenStackInfrastructure/utils.py

```diff
@@ -313,4 +313,14 @@ def getNetSubnetsGws_from_GwInfo(external_gateway_info):
 
     return (network, subnets, gateways)
 
+def isValidHostname(hostname):
```

Comment:
This seems unnecessarily strict to me.  I'm not really clear on why we need it at all.  (see previous comment)